server/middlewares/checkUser.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Resolve this issue. Consult the eslint official documentation.

Reply:
Already resolved